sqlparser_bench/benches/sqlparser_bench.rs

```diff
@@ -23,24 +23,24 @@ fn basic_queries(c: &mut Criterion) {
     let mut group = c.benchmark_group("sqlparser-rs parsing benchmark");
     let dialect = GenericDialect {};
 
-    let string = "SELECT * FROM table WHERE 1 = 1";
+    let string = "SELECT * FROM my_table WHERE 1 = 1";
```

Comment:
"table" is a keyword so you can't use it as a table name without putting it in quotes